src/examples/ClipboardExamplePage.tsx

```diff
@@ -43,16 +43,22 @@ export const ClipboardExamplePage: React.FunctionComponent<{}> = () => {
       <Example title="Copy text to the Clipboard." code={example1jsx}>
         <View style={{alignItems: 'flex-start', gap: 12}}>
           <Button
+            accessibilityLabel="Example copy text"
```

Comment:
Why doesn't Button.title become the default accessibilityLabel?

Reply:
I think it's because of how we define the accessibilityLabel property [here](https://github.com/microsoft/react-native-windows/blob/63d919e36e0d6ad93f1e772df53a1e7c1a5aa1da/vnext/src-win/Libraries/Components/Button.windows.js#L378). 

If we set the `accessibilityLabel` property based on the `title` instead would probably automatically do that. If the `title` property contains text that is the same as the ControlType (which would be 'button' here), then it's up to the client to add an appropriate `accessibilityLabel` property. Can open up a PR with this change if this approach makes sense.

Reply:
It seems to match the cross-platform implementation:
https://github.com/facebook/react-native/blob/a6f5e5adebed3d9da411f99548e2d8ce96636e16/packages/react-native/Libraries/Components/Button.js#L366

It seems like a bad default to me personally... but we're matching core and that's what we should do. As in this is the same JS you'd use on mobile. It's just... duplicative and verbose. Oh well.